pkg/commands/apply.go

```diff
@@ -82,6 +82,11 @@ func addApply(topLevel *cobra.Command) {
   ko apply -f config -- --namespace=foo --kubeconfig=cfg.yaml
 `,
 		RunE: func(cmd *cobra.Command, args []string) error {
+			err := options.ValidateOptions(po, bo)
```

Comment:
style nit: if the err isn't used outside of the `if` block, I tend to prefer:

```
if err := options.Validate(...); err != nil {
  return fmt.Errorf(...)
}
```

This can make it clearer to readers that nothing else uses the `err`, since it won't even be defined outside the `if` block.